r4e-image.sh

```diff
@@ -74,29 +78,33 @@ composer-cli blueprints push $BLUEPRINT_FILE
 # create image
 echo "creating image $IMAGE_NAME"
 composer-cli compose start $IMAGE_NAME rhel-edge-commit > temp.out
-BUILD_ID=`cat temp.out | awk '{print $2}'`
+BUILD_ID=$(awk '{print $2}' temp.out)
```

Comment:
out of curiosity, what is the advantage or difference between the two?

Reply:
There is spare action (or additional process execution) when you first `cat` a file and then invoke awk.
It is useful when processing a file which is being written to (e.g. trying to filter only ERROR from a live log file).

Reply:
well...in our case we don't filter a log file
it is shell. we execute many commands/processes.